packages/query/src/queriers/compact-block.ts

```diff
@@ -17,16 +11,10 @@ export class CompactBlockQuerier implements CompactBlockQuerierInterface {
     this.client = createClient(grpcEndpoint, CompactBlockService);
   }
 
-  async *compactBlockRange({
-    startHeight,
-    keepAlive,
-    abortSignal,
-  }: CompactBlockRangeParams): AsyncIterable<CompactBlock> {
-    const req = new CompactBlockRangeRequest({ keepAlive, startHeight });
-    const iterable = this.client.compactBlockRange(req, { signal: abortSignal });
-    for await (const res of iterable) {
-      if (!res.compactBlock) throw new Error('No block in response');
-      yield res.compactBlock;
-    }
+  compactBlockRange(
+    req: { startHeight?: bigint; keepAlive?: boolean },
```

Comment:
what's generally the distinction here between the former which was an async generator function that yielded `CompactBlock` objects from a `CompactBlockRangeRequest`, versus this construction that directly returns an `AsyncIterable` of `CompactBlockRangeResponse` objects. Aren't they functionally the same?

Reply:
i'm not sure, but it the former call seemed to delay processing of the stream for some reason.

i'm skeptical of the need for these queriers in general - and mostly they just unwrap one layer of an object, or catch errors. some of them are very thin like this.